corehq/apps/ota/tests/digest_restore.py

```diff
@@ -5,6 +5,13 @@
 from django_digest.test import Client
 
 
+def setupModule():
+    from unittest import SkipTest
+    raise SkipTest("this is broken, hasn't been run by django test runner. Error: "
```

Comment:
@sravfeyn don't worry about these tests, they'll be fixed in this PRhttps://github.com/dimagi/commcare-hq/pull/11633